rubin_scheduler/utils/consdb.py

```diff
@@ -316,8 +316,32 @@ def consdb_visits(self) -> pd.DataFrame:
         prior_day_obs_date = day_obs_date - timedelta(days=self.num_nights)
         prior_day_obs_int = int(prior_day_obs_date.strftime("%Y%m%d"))
 
+        # To avoid duplicate columns, explicitly include list of columns to
+        # to return in the query instead of using a "SELECT *".
+        def columns_query(table, instrument):
+            return f"""
+                SELECT column_name FROM information_schema.columns
+                WHERE table_name = '{table}'
+                  AND table_schema = 'cdb_{instrument}'
+            """
+
+        exposure_columns = (
+            query_consdb(columns_query("exposure", self.instrument)).loc[:, "column_name"].values
+        )
+        ql_columns = (
+            query_consdb(columns_query(f"visit{self.num_exposures}_quicklook", self.instrument))
+            .loc[:, "column_name"]
+            .values
+        )
+        visit_query_exposure_columns = ", ".join([f"e.{c} AS {c}" for c in exposure_columns])
+        visit_query_ql_columns = ", ".join(
+            [f"v{self.num_exposures}q.{c} AS {c}" for c in ql_columns if c not in exposure_columns]
+        )
+        visit_query_columns = ", ".join([visit_query_exposure_columns, visit_query_ql_columns])
```

Comment:
We should standardize on using "visit1" table OR exposure table, but once choosing one, should name the subsequent use of those values match. 
Really, visit1 goes with visit1_quicklook, and matches the term "visits" so I'd prefer going with the visit1 table instead of exposure.
Exposure really is exposures .. if /when we do multiple snaps per visit, exposure will deviate significantly from visit1.

Reply:
Okay, yeah, I'm switching everything to visit1. It's visits we really care about, because the scheduler doesn't schedule exposures independently of visits.